src/strings/nfg.c

```diff
@@ -471,12 +471,25 @@ void MVM_nfg_init(MVMThreadContext *tc) {
     cache_crlf(tc);
 }
 
+static void nfg_trie_node_destroy(MVMThreadContext *tc, MVMNFGTrieNode *node) {
+    MVMint32 i = 0;
```

Comment:
Nit-pick, but in some of the other commits you put the declaration right in the first entry of the for.